src/Styles/StyledComponents.js

```diff
@@ -2,14 +2,28 @@ import styled from "styled-components";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 
 // colors*********************************
-const primaryLight = "#ffffff";
-const textPrimary = "#000000";
 const primary = "#fafafa";
+const primaryLight = "#ffffff";
+// const primaryDark = "#c7c7c7";
+
 const secondary = "#00bcd4";
+// const secondaryDark = "#008ba3";
+
+const textPrimary = "#000000";
 
 //text************************************
 export const Text = styled.p`
-  color: #00000099;
+  color: #666666;
+`;
+
+export const Bold = styled.h3`
+  color: #212121;
+`;
+
+export const TextContainer = styled.div`
+  max-width: 700px;
+  margen-left: auto;
```

Comment:
I don't think this is working as intended it should be `margin-left`, `margin-right`